source/PlayState.hx

```diff
@@ -1,19 +1,131 @@
 package;
 
+import flixel.FlxG;
 import flixel.FlxState;
+import flixel.group.FlxGroup.FlxTypedGroup;
+import flixel.text.FlxText;
 
 class PlayState extends FlxState
 {
+	/** Starting X coordinate of ControllableHero sprite.**/
+	static var CHXPOS:Float = 320;
+
+	/** Starting Y coordinate of ControllableHero sprite.**/
+	static var CHYPOS:Float = 300;
+
+	/** X coordinate of Haxe logo hero.**/
+	static var HEROXPOS:Float = 300;
+
+	/** Y coordinate of Haxe logo hero.**/
+	static var HEROYPOS:Float = 300;
+
+	/** X coordinate of text.**/
+	static var TEXTXPOS:Float = 235;
+
+	/** Y coordinate of text.**/
+	static var TEXTYPOS:Float = 200;
+
+	/** The displayed text.**/
+	static var TEXTTEXT:String = "Salvete, omnes!";
+
+	/** The size of the text.**/
+	static var TEXTSIZE:Int = 24;
+
+	/** The minimum starting number of ScreensaverHero sprites.**/
+	static var NUMSSHEROMIN:Int = 10;
+
+	/** The maximum starting number of ScreensaverHero sprites.**/
+	static var NUMSSHEROMAX:Int = 20;
+
+	/** The X coordinate the ScreenSaverHero sprites start.**/
+	static var SSHXPOS:Float = 320;
+
+	/** The Y coordinate the ScreenSaverHero sprites start.**/
+	static var SSHYPOS:Float = 180;
+
+	/**The size of the bounding box. This is a percentage of the play screen
+		in decimal (e.g. .75 or .5).**/
+	static var BOXSIZE:Float = .75;
+
+	/** How thick the walls of the bounding box are.**/
+	static var WALLWIDTH:Int = 5;
+
+	// Class level variables for collision detection.
+	var controllableHero:ControllableHero;
+	var screenSaverHeroGroup = new FlxTypedGroup<ScreenSaverHero>();
+	var wallGroup = new FlxTypedGroup<Wall>();
+
 	override public function create()
 	{
 		super.create();
+		controllableHero = new ControllableHero(CHXPOS, CHYPOS);
 
-		var myHero = new Hero(300, 300);
+		// Static Haxe logo Hero.
+		var myHero = new Hero(HEROXPOS, HEROYPOS);
 		add(myHero);
+
+		var text = new flixel.text.FlxText(TEXTXPOS, TEXTYPOS, 0, TEXTTEXT, TEXTSIZE);
+		text.color = 0x0080FF;
+		text.font = "Time New Roman";
+		add(text);
+
+		// Generates random number of ScreenSaverHero sprites facing and moving
+		// in different directions.
+		var numSSHeroes = FlxG.random.int(NUMSSHEROMIN, NUMSSHEROMAX);
+		for (i in 0...numSSHeroes)
+		{
+			var screenSaverHero = new ScreenSaverHero(SSHXPOS, SSHYPOS);
+			screenSaverHeroGroup.add(screenSaverHero);
+			add(screenSaverHero);
+		}
```

Comment:
This is another good example of how we could adjust the code itself to be a bit more self-documenting. 

The comment is definitely helpful in this case, but the fact that it's helpful shows that the code on its own is kinda hard to read and interpret.

In a case like this, where we have a block of code with a focused purpose (which we made a comment to explain) it's a good indicator that the block of code would make sense as its own separate private helper method.

If we made a method like this:
```Haxe
private function generateRandomizedScreenSaverHeroes() {
    var screenSaverHeroCount = FlxG.random.int(SCREEN_SAVER_COUNT_MIN, SCREEN_SAVER_COUNT_MAX);
    
   for (i in 0...screenSaverHeroCount) {
        var screenSaverHero = new ScreenSaverHero(SCREEN_SAVER_START_X, SCREEN_SAVER_START_Y);
        screenSaverHeroGroup.add(screenSaverHero);
        add(screenSaverHero);
    }
}
```

Then we could just drop a call to this new method `generateRandomizedScreenSaverHeroes()` into this spot, and we wouldn't have much need for a comment - the code directly says that it's generating randomized screen saver heroes just through the method name.

Reply:
To clarify, no change needed, these are things to consider for future projects.